app/pb_kits/playbook/pb_date_picker/_date_picker.jsx

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+
+import React, { useEffect } from 'react'
+import classnames from 'classnames'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
+import { globalProps } from '../utilities/globalProps.js'
+// import flatpickr from 'flatpickr'
+import { Body, Caption, Icon } from '../'
+import datePickerHelper from './date_picker_helper.js'
+
+type DatePickerProps = {
+  aria?: object,
+  className?: String,
+  data?: object,
+  defaultDate?: String,
+  disableDate?: Array,
+  disableRange?: Array,
+  disableWeekdays?: Array,
+  error?: String,
+  format?: String,
+  hideLabel?: Boolean,
+  id?: String,
+  label?: String,
+  maxDate: String,
+  minDate: String,
+  mode?: String,
+  name: String,
+  pickerId?: String,
+  readOnly?: Boolean,
+  yearRange?: Array,
+}
+const DatePicker = (props: DatePickerProps) => {
+  const {
+    aria = {},
+    className,
+    data = {},
+    defaultDate = '',
+    disableDate = null,
+    disableRange = null,
+    disableWeekdays = null,
+    error,
+    format = 'm/d/Y',
+    hideLabel = false,
+    id,
+    label = 'Date Picker',
+    maxDate,
+    minDate,
+    mode = 'single',
+    name,
+    pickerId,
+    readOnly = false,
+    yearRange = [ 1900, 2100 ],
+  } = props
+
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+  const classes = classnames(
+    buildCss('pb_date_picker_kit'),
+    className,
+    globalProps(props),
+    error ? 'error' : null,
+  )
+
+  useEffect(() => {
+    datePickerHelper({
+      defaultDate: defaultDate,
+      disableDate: disableDate,
+      disableRange: disableRange,
+      disableWeekdays: disableWeekdays,
+      format: format,
+      maxDate: maxDate,
+      minDate: minDate,
+      mode: mode,
+      pickerId: pickerId,
+      propModel: 'React',
+      readOnly: readOnly,
+      yearRange: yearRange,
+    })
+  }, [])
+
+  return (
+    <div
+        {...ariaProps}
+        {...dataProps}
+        className={classes}
+        id={id}
+    >
+      {className}
+      <If condition={!hideLabel}>
+        <Caption
+            text={label}
+        />
+      </If>
+      <div className="input_wrapper">
+        <input
+            autoComplete="off"
+            id={pickerId}
+            name={name}
+        />
```

Comment:
Many of our form kits allow the user to bring their own input as they might need additional things added to the input that we don't enable through props. Do you think we would need that for this kit?

Reply:
Follow up...do you think we could use the text input kit here instead?

Reply:
Removed html input tag and used text input kit instead.  Dark is now passed as a prop to the text input kit.